src/pages/learn/html5/tutorials/tanks-canvas-socketio.mdx

```diff
@@ -69,7 +69,7 @@ Run the following command to setup your project:
   ```
   </code>
   <code title="CMD">
-    ```batch
+    ```bash
```

Comment:
I think this should remain `batch` unless there isn't formatting for windows `batch` scripts, or if this just looks better as `bash`. Just want to make sure this isn't a case of mistaking it for a misspelling 💯